### PR TITLE
add style

### PR DESCRIPTION
**Pull request by ReemMohamedAbdelfatah**
In this pull request the following is done:

- Add required styles .

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -6,17 +6,56 @@ import { removeBook } from '../Redux/books/books';
...
+        </div>
+        <div className="progress-divider" />
+        <div className="current-chapter">
+          <p className="current-chapter-title">CURRENT CHAPTER</p>
+          <p className="current-chapter-number">Chapter 17</p>
+          <button type="button" className="btn-update">
+            UPDATE PROGRESS
+          </button>
+        </div>
+      </div>
```
Comment (danmainah) [inline src/components/Book.js:58]: - [x] @ReemMohamedAbdelfatah kindly check on your styling the font  differs from the given design, also the color of the navbar items home and categories differs from the design given, check the comparison below for better understanding:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image  | image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ReemMohamedAbdelfatah , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️

- [x] @ReemMohamedAbdelfatah  kindly ensure your links in the readme file are working, currently, link to your LinkedIn profile also live link is not working , in case you do not have a live link you can state coming soon or remove the live link section

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Book.js (hunk 1 above)
Comment (ReemMohamedAbdelfatah) [reply src/components/Book.js:58]: @danmainah Done👌

**Review by roshan-bajgain — APPROVED** [review]
Hi @ReemMohamedAbdelfatah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ReemMohamedAbdelfatah** [comment]
Thanks!

**Pull request merged**